### ISSUE TITLE
OTG and powering Raspberry Pi 4 via GPIO pins

### ISSUE THREAD

**Issue by etkaar** [pikvm/pikvm#940]
Is it possible powering the Raspberry Pi 4 via the GPIO pins while using the USB-C port only for OTG?

**Comment by mdevaev** [comment]
Yes, but you need to cut power line in Type-C cable to prevent backpowering.
